### PR TITLE
{lib}[GCCcore/8.3.0] LibTIFF v4.1.0

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/ae2d8e96ac0c504a3efac547c8cf4284 for a full test report.

**Comment by verdurin** [comment]
@furstj Please submit a test build

**Comment by furstj** [comment]
Test report by @furstj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
pc-12101-fuji - Linux UNKNOWN UNKNOWN, Intel(R) Xeon(R) CPU E3-1241 v3 @ 3.50GHz, Python 3.6.9
See https://gist.github.com/2ac335135364198872eeb09a11b29f5c for a full test report.

File: easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+easyblock = 'ConfigureMake'
+
+name = 'LibTIFF'
+version = '4.1.0'
+
+homepage = 'https://www.remotesensing.org/libtiff/'
```
Comment (verdurin) [inline easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-8.3.0.eb]: @furstj Are you sure about this `homepage`? For me it points to a domain parking page.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3135.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c8f7d82b7daf1b91169279ea8cfcffc4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/c7bb27f9abdce66ccc36810ba822a33b for a full test report.

**Comment by boegel** [comment]
Going in, thanks @furstj!

**Pull request merged**